ops/terraform/setup/main.tf

```diff
@@ -0,0 +1,44 @@
+###########################################################################################
+#
+# This file creates the bare minimum infrastructure to start storing remote state.
+# It can't store its own remote state, so this file contains only one resource.
+#
+# In other words, do not apply this file multiple times, as it will fail due to lack of
+# state - it won't know it already created the resources.
+#
+###########################################################################################
+
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=3.23.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  use_oidc = true
+  features {}
+}
+
+# Note: Only lowercase and alphanumeric characters allowed for name
+resource "azurerm_storage_account" "storage_account" {
+  name                      = "${var.project}tfstate${var.env}"
```

Comment:
Are you thinking of creating a new SA each time a new environment is created? Currently we are storing all our state files dynamically in a global SA called `tfstaterv2024` under the `reportvision-rg` Resource Group.